server/controllers/allocationsController.ts

```diff
@@ -370,6 +379,7 @@ export default class AllocationsController {
         emailCopyOptOut: form.emailCopyOptOut === 'yes',
       })
     )
+    // const laoCase = await this.allocationsService.getLaoStatus(crn, res.locals.user.token)
```

Comment:
commented out?

Reply:
..commented out for now, will be required for future ticket